### PR TITLE
Theme Slate

### PR DESCRIPTION
**Pull request by BecAlenvers**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -1,2 +1,7 @@
-theme: jekyll-theme-minimal
+
+
+title: Šijeme roušky pro Brno a okolí
```
Comment (LucieKarm) [inline _config.yml]: @BecAlenvers ```Face mask sewing for hospitals, social workers and carers``` ???

File: _layouts/default.html [hunk 2]
```diff
@@ -1,38 +1,63 @@
...
-    <script src="//html5shiv.googlecode.com/svn/trunk/html5.js"></script>
-    <![endif]-->
   </head>
+
   <body>
-      {{ content }}
-  <p>Autor: <a href="https://www.linkedin.com/in/lucie-karmova/">Lucie Karmova</a></p>
-  <p><small>Hosted on GitHub Pages &mdash; Theme by <a href="https://github.com/orderedlist">orderedlist</a></small></p>
-  <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/">
-      <img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-nd/4.0/88x31.png" />
```
Comment (LucieKarm) [inline _layouts/default.html:21]: @BecAlenvers Why removing CC license?

File: _layouts/default.html [hunk 3]
```diff
@@ -1,38 +1,63 @@
...
+    </div>
+
+    {% if site.google_analytics %}
+      <script>
         (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
         (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
         m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-        })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
```
Comment (LucieKarm) [inline _layouts/default.html]: @BecAlenvers Is this really O.K.?  
```
- https://www
+ //www
```

**Review by LucieKarm — CHANGES_REQUESTED** [review]
Perfect! Just three small things :heart:

**Comment by BecAlenvers** [comment]
Thanks for the review!

**Review by LucieKarm — APPROVED** [review]

**Pull request merged**